DNN Platform/Library/UI/Containers/ActionBase.cs

```diff
@@ -26,13 +21,16 @@ namespace DotNetNuke.UI.Containers
     /// </remarks>
     public abstract class ActionBase : UserControl, IActionControl
     {
-        protected bool m_supportsIcons = true;
-        private ActionManager _ActionManager;
-        private ModuleAction _ActionRoot;
+        private bool supportsIcons = true;

```

Comment:
There's no evidence that this is in use (via [a GitHub search](https://github.com/search?q=m_supportsIcons&type=Code)), but this change from `protected` to `private` is a breaking change.  I've never used `ActionBase`, so I'm not sure how likely it is that someone would make use of this field.

Reply:
Well since it was protected and not used from anywhere else in this library, stylecop said it should be private. I can revert it if you think it may be a breaking change, but a third party would already not have access to a protected member right ?

Reply:
Oh, wait, a derived class could access it, let me change that back then...

Reply:
Ok, been addresses in https://github.com/dnnsoftware/Dnn.Platform/pull/4224/commits/e790b39f3b48f9acad819e373172f00639c1288f